### PR TITLE
improvement in project search

### PR DESCRIPTION
**Pull request by NoemiNahomy**
Improvement in project search
api : /api/v2/projects/?campaign=4+County+Digitization+Liberia

Also to improve pagination response times in project search, the [seq_page_cost](https://www.postgresql.org/docs/9.5/runtime-config-query.html#GUC-SEQ-PAGE-COST) parameter could be set in postgres.

### CODE REVIEW

File: backend/services/project_search_service.py [hunk 1]
```diff
@@ -325,7 +328,13 @@ def _filter_projects(search_dto: ProjectSearchDTO, user):
 
         all_results = []
         if not search_dto.omit_map_results:
-            all_results = query.all()
+            query_result = query
+            query_result.column_descriptions.clear()
```
Comment (ramyaragupathy) [inline backend/services/project_search_service.py:332]: Can you please explain about these steps a bit?

**Comment by ramyaragupathy** [comment]
@NoemiNahomy - Any changes that we have on the DB, let's do it via a migration.

File: backend/services/project_search_service.py (hunk 1 above)
Comment (NoemiNahomy) [reply backend/services/project_search_service.py:332]: in this line when we make this query `query.all()`, this query uses all the project attributes and the filter and we don't need all those attributes, it only needs 3. So I use the filter but I only do the query with 3 values.       
These values ​​are used for the geojson of the map
 https://github.com/hotosm/tasking-manager/blob/2e94a550a28b6d9bff2be8953d69ea693fa04b25/backend/services/project_search_service.py#L193

**Comment by ramyaragupathy** [comment]
DB setting will be handled in a separate  PR migration

**Pull request merged**